app/assets/v2/css/ios.css

```diff
@@ -0,0 +1,188 @@
+
+.ios { 
+  background: #fefefe;
+}
+.ios .header{
+  background: none;
+}
+
+.ios-hero__container .nav-link:visited .nav-link--active,
+.ios-hero__container .nav-link--active {
+  background: #FFF;
+  border-radius: .5rem;
+  color: #8E2ABE !important;
+}
+
+.ios-hero__container .nav-link--active:hover {
+  color: #FFF !important;
+}
+
+.ios-hero__container {
+  background: -webkit-linear-gradient(138deg, #8E2ABE 0%, #37EA9C 100%);
+  background: -o-linear-gradient(138deg, #8E2ABE 0%, #37EA9C 100%);
+  background: linear-gradient(228deg, #8E2ABE 0%, #37EA9C 100%);
+  height: 750px;
+  min-height: 750px;
+  margin-bottom: 225px;
+}
+
+.ios-hero__container .header-line p{
+  font-family: Futura;
+  font-size: 2em;
+}
+.ios-hero__container .header-line {
+  letter-spacing: normal;
+  line-height: normal;
+  font-size:3rem;
+  font-style: normal;
+  padding-top: 150px;
+}
+.ios-hero__container .header-line h3 {
+  font-size: 1.75rem;
+}
+
+.ios .btn-outline-info {
+  color: white;
+  background: -webkit-linear-gradient(138deg, #37EA9C 0%, #8E2ABE 100%);
+  background: -o-linear-gradient(138deg, #37EA9C 0%, #8E2ABE 100%);
+  background: linear-gradient(228deg, #37EA9C 0%, #8E2ABE 100%);
+  border: 3px solid;
+  border-radius: .5rem;
+}
+
+.ios .footer {
+  border-top: 1px solid #eff3ee;
+  background: #fff;
+  padding: 0 15px;
+}
+
+.ios .footer .nav-link {
+  color: black !important;
+}
+
+.ios .footer img {
+  filter: none;
+}
+
+.ios .video {
+    width: 100%;
```

Comment:
inconsistent indentation

Reply:
https://github.com/gitcoinco/web/pull/207/commits/fd67299681aed5fc61b9f08a4a608926920e1620 fixes this issue.